sdk/Lusid.Sdk.Tests/tutorials/Instruments/EquityExamples.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Tests.Utilities;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.Tutorials.Instruments
+{
+    [TestFixture]
+    public class EquityExamples: DemoInstrumentBase
+    {
+        [Test]
+        public void EquityCreationAndUpsertionExample()
```

Comment:
this needs a unique LusidFeature from this needs a unique LusidFeature (https://gitlab.com/finbourne/salesengineering/lusid-features/-/blob/master/features.yml)

Reply:
You beat me to the chase, I was going to post and ask. I am allowed/good to add to that yml file myself? 
Is there a logic in how do I allocate the codes?

Reply:
yes raise a PR, they need to be unique, the ones you are interested in are most like going to be F5-x or F22-x